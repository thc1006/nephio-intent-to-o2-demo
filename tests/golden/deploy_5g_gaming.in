Deploy a 5G network slice with ultra-low latency for gaming services at edge site 1