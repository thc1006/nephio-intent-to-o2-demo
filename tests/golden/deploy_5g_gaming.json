{
  "intentId": "intent_gaming_edge1",
  "name": "5G Gaming Slice - Edge Site 1",
  "description": "Deploy a 5G network slice with ultra-low latency for gaming services at edge site 1",
  "targetSite": "edge1",
  "parameters": {
    "type": "5G_network_slice",
    "requirements": {
      "latency": "5ms",
      "bandwidth": "100Mbps",
      "reliability": "99.99%"
    },
    "configuration": {
      "service": "gaming",
      "qos": "ultra-low-latency",
      "sliceType": "eMBB"
    }
  },
  "priority": "high",
  "lifecycle": "draft"
}